documentation/connect_puppet_master.markdown

```diff
@@ -14,7 +14,7 @@ canonical: "/puppetdb/latest/connect_puppet_master.html"
 [report]: ./api/query/v3/reports.html
 [store_report]: ./api/commands.html#store-report-version-1
 [report_format]: ./api/wire_format/report_format_v3.html
-[url_prefix_setting]: ./configure.html#url-prefix
+[terminus_failover]: ./terminus_failover.html
```

Comment:
Good catch, that was my old name for puppetdb_connection, before I realized there was a puppetdb_conf file. This should be removed.